### ISSUE TITLE
Missing text/icon in custom toolbar buttons

### ISSUE THREAD

**Issue by webismymind** [tinymce/tinymce#2466]
Here is the code used to initialize TinyMCE and its toobar:

```javascript
tinymce.init({ 
	theme: "modern",
	plugins: [  "pagebreak", "textcolor", "table", "searchreplace", "image" ],
	toolbar1:  "undo redo | fontselect fontsizeselect formatselect | bold italic underline strikethrough forecolor backcolor | bullist numlist | alignleft aligncenter alignright alignjustify | outdent indent | pagebreak"
});
```
On Google Chrome (MacOS), the toolbar is OK:
![Result on Google Chrome (MacOS)](https://www.dropbox.com/s/rdfpvpc66dpp99l/Chrome.png?raw=1&dl=0)

On Firefox (MacOS), the "fontselect", "fontsizeselect" and "formatselect" are empty:
![Result on Firefox (MacOS)](https://www.dropbox.com/s/mrg2sij8fuf5ili/FF.png?raw=1)

Note that if I don't specify any custom toolbar, then the toolbar is OK both on Firefox and Chrome.

**Comment by spocke** [comment]
Can this be reproduced on the tinymce site or in a fiddle.tinymce.com could be a css conflict?

**Comment by webismymind** [comment]
Thanks for your answer. 

I tried to reproduce on fiddle.tinymce.com (which I should have done in the first place), but everything was working well.

So I tried again in our application and the problem has disappeared! All toolbar buttons are now visible and the toolbar has the same "flat" look as in Google Chrome.

I don't know if something has changed on our side (I don't think so) or if the upgrade to Firefox 42.0 fixed this issue. Anyway, sorry for bothering and thanks again.
